client/app/main/page.js

```diff
@@ -3,18 +3,48 @@ import TopBar from '../topbar';
 import Chat from './chat/chat';
 import Renderer from './renderer/renderer';
 import Editor from './editor/editor';
-import { Box, Button } from '@mui/material';
+import { Box, Button, FormControl, InputLabel, Select, MenuItem } from '@mui/material';
 import { useState } from 'react';
 
 export default function Page() {
   const [showChat, setShowChat] = useState(true);
   const [html, setHtml] = useState('<h1> Renderer </h1>');
   const [css, setCss] = useState('');
   const [js, setJs] = useState('');
+  const [language, setLanguage] = useState('html');
```

Comment:
Css, and js wont be needed, since it returns pure html. Language can probably be set to framework, and maybe another state for styling library